src/test/regress/sql/multi_extension.sql

```diff
@@ -175,3 +175,131 @@ ALTER EXTENSION citus UPDATE;
 
 -- if cache is invalidated succesfull, this \d should work without any problem
 \d
+
+\c - - - :master_port
+
+CREATE VIEW table_fkeys AS
+SELECT r.conname AS "Constraint",
+       pg_catalog.pg_get_constraintdef(r.oid, true) AS "Definition",
+       conrelid AS "relid"
+FROM pg_catalog.pg_constraint r
+WHERE r.contype = 'f';
```

Comment:
OK, changing a previous `UNIQUE INDEX` to a `UNIQUE CONSTRAINT` makes my `VIEW` work again.

Still, since the initial implementation only read the names and OIDs of the check and foreign-key constraints, I think the risk of using the `pg_catalog` tables is low.